.github/workflows/upgrade-cdktf.yml

```diff
@@ -0,0 +1,109 @@
+name: upgrade-cdktf
+on:
+  schedule:
+    - cron: 13 */6 * * *
+  workflow_dispatch: {}
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+jobs:
+  check_versions:
+    name: Check CDKTF versions
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Install
+        run: yarn install
+      - name: Get current CDKTF version
+        id: current_version
+        # NOTE: This uses the version in projenrc.template.js as the source of truth (and not the version of cdktf in the repo-manager's package.json)
+        run: |-
+          OLD_VERSION=$(sed -nE 's/cdktfVersion: "\^(.*)",/\1/p' projenrc.template.js | xargs)
+          OLD_VERSION_SHORT=$(cut -d "." -f 2 <<< "$OLD_VERSION")
+          echo "value=$OLD_VERSION" >> $GITHUB_OUTPUT
+          echo "short=$OLD_VERSION_SHORT" >> $GITHUB_OUTPUT
+      - name: Get latest CDKTF version
+        id: latest_version
+        run: |-
+          CDKTF_VERSION=$(yarn info cdktf --json | jq -r '.data.version')
+          CDKTF_VERSION_SHORT=$(cut -d "." -f 2 <<< "$CDKTF_VERSION")
```

Comment:
Does this strip away the patch number? A comment would be useful to remind future me that this is to ignore patch releases.

Reply:
It's not just dropping the patch, what it's doing is grabbing the minor version number from the semver, e.g. if the version is `0.18.0` it results in `18`,   if the version is `1.6.0` it results in `6`. I renamed the variable to `CDKTF_VERSION_MINOR` to be more self-documenting.